src/zope/app/form/browser/tests/test_form.py

```diff
@@ -21,19 +21,31 @@
 from zope.schema.interfaces import ITextLine
 from zope.component import testing
 
-from zope.app.testing import ztapi
+import zope.app.form.testing as ztapi
 
 from zope.app.form.browser import TextWidget
 from zope.formlib.interfaces import IInputWidget
 
+
+from zope.testing import renormalizing
+
+import re
+checker = renormalizing.RENormalizing([
+    (re.compile("u('.*?')"), r"\1"),
+    (re.compile('u(".*?")'), r"\1"),
```

Comment:
Yes 😦 The regex didn't match inside the list/tuple, it only handles an entire line. It was easier to just use print those than to fuss with the regex.